test/topology/conftest.py

```diff
@@ -22,6 +22,12 @@
 from cassandra.connection import DRIVER_VERSION    # type: ignore # pylint: disable=no-name-in-module
 
 
+# Use millisecond format to correlate test.py, test
+# and scylla log events with each other
+logging.basicConfig(
```

Comment:
I think  this, together with the configuration in pytest.ini, will output two messages for each logging call.

You should probably modify pytest.ini instead of this.

Reply:
Fixed.